app/src/main/java/org/wikipedia/feed/view/FeedAdapter.java

```diff
@@ -1,139 +0,0 @@
-package org.wikipedia.feed.view;
-
-import android.content.Context;
-import android.view.View;
-import android.view.ViewGroup;
-
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-import androidx.recyclerview.widget.RecyclerView;
-import androidx.recyclerview.widget.StaggeredGridLayoutManager;
-
-import org.wikipedia.feed.FeedCoordinatorBase;
-import org.wikipedia.feed.accessibility.AccessibilityCard;
-import org.wikipedia.feed.announcement.AnnouncementCardView;
-import org.wikipedia.feed.dayheader.DayHeaderCardView;
-import org.wikipedia.feed.image.FeaturedImageCardView;
-import org.wikipedia.feed.model.Card;
-import org.wikipedia.feed.model.CardType;
-import org.wikipedia.feed.news.NewsCardView;
-import org.wikipedia.feed.offline.OfflineCard;
-import org.wikipedia.feed.offline.OfflineCardView;
-import org.wikipedia.feed.random.RandomCardView;
-import org.wikipedia.feed.searchbar.SearchCardView;
-import org.wikipedia.feed.suggestededits.SuggestedEditsCardView;
-import org.wikipedia.util.DimenUtil;
-import org.wikipedia.views.DefaultRecyclerAdapter;
-import org.wikipedia.views.DefaultViewHolder;
-
-public class FeedAdapter<T extends View & FeedCardView<?>> extends DefaultRecyclerAdapter<Card, T> {
```

Comment:
Somehow if we keep multiple types for the generic type specification, we'll not able to create a `FeedAdapter` in `FeedFragment` with only one type. Because of it, I change it to only extends `View` here.